addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -75,12 +75,28 @@ The Kodi thing supports the following channels:
 | pvr-open-radio   | String    | Opens the PVR Radio channel with the provided name                                                                                                                                           |
 | pvr-channel      | String    | Title of the currently played PVR channel                                                                                                                                                    |
 | shownotification | String    | Shows the provided notification message on the screen                                                                                                                                        |
-| input            | String    | Allows to control Kodi. Valid values are: `Up`, `Down`, `Left`, `Right`, `Select`, `Back`, `Home`, `ContextMenu`, `Info`, `ShowOSD`, `ShowPlayerProcessInfo`, `SendText` and `ExecuteAction` |
+| input            | String    | Allows to control Kodi. Valid values are: `Up`, `Down`, `Left`, `Right`, `Select`, `Back`, `Home`, `ContextMenu`, `Info`, `ShowOSD` and `ShowPlayerProcessInfo`                              |
 | inputtext        | String    | This channel emulates a keyboard input                                                                                                                                                       |
+| inputaction      | String    | Send a predefined action to the Kodi Player to control the UI and/or perform other tasks. Valid values are: `left`, `right`, `up`, `down`, `pageup`, `pagedown`, `select`, `highlight`,      |
+|                                `parentdir`, `parentfolder`, `back`, `menu`, `previousmenu`, `info`, `pause`, `stop`, `skipnext`, `skipprevious`, `fullscreen`, `aspectratio`, `stepforward`, `stepback`, `bigstepforward`,  |
```

Comment:
This won't work. If you click an the `View` button next to this file in GitHub you will see a preview. The table doesn't look nice.

To make @martinvw happy you should format the tables in the `README.md` file.

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table. :+1: